pkgs/by-name/li/livekit-libwebrtc/package.nix

```diff
@@ -74,7 +74,7 @@ let
 in
 stdenv.mkDerivation {
   pname = "livekit-libwebrtc";
-  version = "m114";
```

Comment:
This is a mistake I made in the original PR (sorry!), but package versions must always start with a number (so no "m" prefix like upstream).

Upstream doesn't do tagged releases, but I think it'd still be more fitting to use "125-unstable-2025-03-24" here than the usual "0-unstable-2025-03-24", because the 125 denotes the API compatibility of the release and otherwise it'd be unclear from which branch this commit is coming.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning